cmd/client/cmd/app.go

```diff
@@ -42,85 +47,124 @@ The app name must follow this rules:
 
   With all flags...
   $ teresa app create foo --team bar --cpu 200m --max-cpu 500m --memory 512Mi --max-memory 1Gi --scale-min 2 --scale-max 10 --scale-cpu 70 --process-type web`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		// Getting app name
-		if len(args) != 1 {
-			return newUsageError("You should provide the name of the app")
-		}
-		appName := args[0]
-		// Getting flags
-		team, _ := cmd.Flags().GetString("team")
-		targetCPU, _ := cmd.Flags().GetInt64("scale-cpu")
-		scaleMax, _ := cmd.Flags().GetInt64("scale-max")
-		scaleMin, _ := cmd.Flags().GetInt64("scale-min")
-		cpu, _ := cmd.Flags().GetString("cpu")
-		memory, _ := cmd.Flags().GetString("memory")
-		maxCPU, _ := cmd.Flags().GetString("max-cpu")
-		maxMemory, _ := cmd.Flags().GetString("max-memory")
-		processType, _ := cmd.Flags().GetString("process-type")
-		if team == "" {
-			return newUsageError("You should provide the name of the team to continue")
-		}
-		// creating app params
-		app := models.AppIn{}
-		app.Name = &appName
-		app.Team = &team
-		app.Scale = scaleMin
-		// app rollint update
-		msurge := "30%"
-		munavailable := "30%"
-		app.RollingUpdate = &models.AppInRollingUpdate{
-			MaxSurge:       &msurge,
-			MaxUnavailable: &munavailable,
-		}
-		// app limits
-		cpuString := "cpu"
-		memString := "memory"
-		app.Limits = &models.AppInLimits{
-			Default: []*models.LimitRangeQuantity{
-				&models.LimitRangeQuantity{
-					Resource: &cpuString,
-					Quantity: &maxCPU,
-				},
-				&models.LimitRangeQuantity{
-					Resource: &memString,
-					Quantity: &maxMemory,
-				},
+	Run: createApp,
+}
+
+func createApp(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+	name := args[0]
+
+	team, err := cmd.Flags().GetString("team")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Invalid team parameter: ", err)
+		return
+	}
+	if team == "" {
+		fmt.Fprintln(os.Stderr, "You must provide the name of the team")
+		return
+	}
```

Comment:
what about simplify this validation, like this:
```go
team, err := cmd.Flags().GetString("team")
if err != nil || team == "" {
    fmt.Fprintln(os.Stderr, "You must provide a valid name of the team")
    return
}
```
?
(or just ignore error and only check for blank team name)